omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -1164,24 +1127,22 @@ private Operation createOperation(DelegatingResourceHandler<?> resourceHandler,
 		return searchHandlerDocList;
 	}
 	
-	private String getResourceUrl(String baseUrl, String resourceName) {
-		//Set the root url.
-		return baseUrl + "/v1/" + resourceName;
-	}
-	
-	private boolean hasSearchHandler(String resourceName) {
-		for (SearchHandlerDoc doc : searchHandlerDocs) {
-			if (doc.getResourceURL().contains(resourceName)) {
+	private boolean hasSearchHandler(String resourceName, String resourceParentName) {
```

Comment:
Here goes the actual changes. getParametersListForSearchHandlers(), and getResourceUrl() was unused. So I removed them.

Reply:
Do you mind have separate pull requests?

Reply:
@gayanW ping 😊